test_scripts/Defects/4_5/1885_SDL_does_not_cut_off_unknown_RPC_and_or_param_from_PTU.lua

```diff
@@ -0,0 +1,158 @@
+---------------------------------------------------------------------------------------------------
+-- Issue: https://github.com/SmartDeviceLink/sdl_core/issues/1885
+-- Precondition:
+-- 1. SDL and HMI are started.
+-- 2. App is registered and activated
+-- Steps:
+-- 1. In case SDL received UpdatedPT with at least one <unknown_parameter> or <unknown_RPC>
+-- Expected result:
+-- SDL must cut off <unknown_parameter> or <unknown_RPC> continue validating received PTU without <unknown_parameter> or
+-- <unknown_RPC> merge valid Updated PT without <unknown_parameter> or <unknown_RPC> with LocalPT in case
+-- of no other failures
+-- Actual result:N/A
+---------------------------------------------------------------------------------------------------
+--[[ Required Shared libraries ]]
+local runner = require('user_modules/script_runner')
+local common = require("user_modules/sequences/actions")
+
+--[[ Test Configuration ]]
+runner.testSettings.isSelfIncluded = false
+
+--[[ Local variables ]]
+-- set default parameters for 'SendLocation' RPC
```

Comment:
Why we need params for Sendlocation?

Reply:
@GetmanetsIrina , These params are necessary for successful SendLocation RPC processing after PTU with unknown RPC